vizro-core/changelog.d/20240902_101331_huong_li_nguyen_enable_header_footer.md

```diff
@@ -0,0 +1,48 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Added
+
+- Enable `header` and `footer` argument inside `Graph`, `Table` and `AgGrid`. ([#669](https://github.com/mckinsey/vizro/pull/669))
+- Enable `title` argument inside `Graph`. ([#669](https://github.com/mckinsey/vizro/pull/669))
+
+<!--
+### Changed
+
+- A bullet item for the Changed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
```

Comment:
Hmm.. not sure if it's worth doing as there is a warning returned already. I can add it if you guys prefer it, but I am indifferent here 😄 

@petar-qb @maxschulz-COL

Reply:
I think all Vizro users somewhere in their app have a Graph.figure.title, and will probably try to find what's the reason behind the "strange" warnings after Vizro is updated. I guess that their next move will be searching for Vizro changelog file and trying to find out what's happening. So, as this will hit all (or almost all) Vizro users, I think we should mention this change to the changelog, even though we don't prefer mentioning minor styling changes.

Any strong opinion on this @maxschulz-COL?

Reply:
If you have a strong preference for it, let me just add it in :)

Reply:
Lets do it!